cobra/cmd/license_lgpl.go

```diff
@@ -0,0 +1,188 @@
+package cmd
+
+func initLgpl() {
+	Licenses["lgpl"] = License{
+		Name:            "GNU Lesser General Public License",
+		PossibleMatches: []string{"lgpl", "lesser gpl", "gnu lgpl"},
```

Comment:
Thank you for your remark! I don't know much about licenses.
So I will use LGPL 2.1 as it good for LGPL such as for lesser GPL. Or it's not right?